### PR TITLE
Add a Scala tutorial

### PR DESCRIPTION
**Pull request by julienrf**
This tutorial contents is based on the Scala MOOCs (on Coursera) (cf https://github.com/scala-exercises/scala-exercises/issues/603)

Can someone rename this repository to `exercises-scalatutorial`?

### CODE REVIEW

**Comment by juanpedromoreno** [comment]
Hey @jdesiloniz  @julienrf !

Is this PR ready to be merged? Thanks!

**Comment by julienrf** [comment]
Hi Juan, thanks for asking! I would wait a little bit before merging because we might polish a few things. Our task queue has several other tasks with a higher priority but we will go back to this one soon.

**Comment by heathermiller** [comment]
This looks really nice to me! Is it ready to merge yet @julienrf? What else would you like to do?

**Comment by julienrf** [comment]
We can merge it.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,45 @@
 
 This repository hosts a library for the first course of the [Scala MOOC](https://www.coursera.org/specializations/scala) ("Functional Programming Principles in Scala").
 
+## Run Locally
+
+- Clone this repository, compile and publish the project:
+
+~~~ sh
+git clone git@github.com:scalacenter/exercises-fpprinciples.git
```
Comment (heathermiller) [inline README.md]: Seems this would need to change before merge. Same for directory below, etc.

Comment (julienrf) [inline README.md]: Fixed.

**Comment by julienrf** [comment]
@jdesiloniz @juanpedromoreno I think only you have the right to merge it :)

**Comment by juanpedromoreno** [comment]
Thanks!

**Comment by raulraja** [comment]
@julienrf @heathermiller I made you both admins in this repo in case you want to moderate or modify its content. We are beyond excited for this PR, thanks so much for your time and this PR.

**Comment by MaureenElsberry** [comment]
Hi, @heathermiller and @julienrf - I sent you both an email regarding our announcement for when this library is released on the platform for a quick review before we publish. Let me know if you want anything changed! And this is awesome, great work.

**Pull request merged**